python/ccf/clients.py

```diff
@@ -174,6 +174,13 @@ def get_curve(ca_file):
     )
 
 
+def unpack_seqno_or_view(data):
+    (value,) = struct.unpack("<q", data)
+    if value == -1:
```

Comment:
`seqno` default is `kv::NoVersion` (`std::numeric_limits<Version>::min()`) while `view` default is `-1`. I think each of those should have a different path for these edge-case values.

Reply:
Why not make both -1?

Reply:
I changed kv::NoVersion to be -1

Reply:
As pointed by @jumaffre separately, we can't do that because we do use negative versions internally, so we should use `std::numeric_limits<Version>::min()` for both.